app/views/items/show.html.erb

```diff
@@ -1,3 +1,17 @@
+<% content_for :head do %>
+  <meta property="og:title" content="<%= page_attr(:title) %>">
+  <meta property="og:url" content="<%= request.original_url %>">
+
+  <% if @item.description.present? %>
+    <meta name="description" content="<%= @item.description.to_plain_text %>">
+    <meta property="og:description" content="<%= @item.description.to_plain_text %>">
+  <% end %>
+
+  <% if @item.image.attached? %>
+    <meta property="og:image" content="<%= item_image_url(@item.image, resize_to_limit: [600, 600]) %>">
```

Comment:
Just saying out loud that I checked to make sure this was a `resize_to_limit` pair that we've already used, so we don't have to generate a new variant just for this use case. Once I get around to #1505 these will all switch to named variants that are preprocessed when the item is saved. Ideally we'll end up with a relatively limited set of sizes to save storage space and processing time.